### PR TITLE
release notes for Aug 21st 2022

### PR DESCRIPTION
**Pull request by allyford**

### CODE REVIEW

**Comment by allyford** [comment]
Addition:
The Open Service Mesh addon has been updated from version 1.1.1 to [version 1.2.0](https://github.com/openservicemesh/osm/releases/tag/v1.2.0) for AKS clusters running 1.24.0+. Please note the breaking changes mentioned in the [version 1.2.0 release notes](https://github.com/openservicemesh/osm/releases/tag/v1.2.0)


Correction:
Missing CWD(Current Working Directory) field in process creation events bug fixed in v1.3.49. Update low level collector image version from 1.3.42 to 1.3.49

Upgrade Azure Disk CSI driver to [v1.19.0](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v1.19.0)

**Comment by IvanAlekseev** [comment]
What are other people thoughts on publishing release notes before releasing to production?

**Comment by PixelRobots** [comment]
> 

The release notes are basically already as it takes some time before they hot Azure regions and then the customer has the choice to do the update or not.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+### Release notes
+
+* Bug fixes
+  * v20220724 contains a bug that will cause clusters running the Open Service Mesh addon to fail when upgrading to v1.24.0, and therefore the upgrade will fail as well. This bug was hotfixed in the v20220731 release. It was also fixed in the v20220807 release but the hotfix needs to rollout to customers already seeing the bug in the mean time. 
+  * Missing CWD(Current Working Directory) field in process creation events fixed. Update low level collector image version from 1.3.42 to 1.3.49
+  * Added a scheduled task "hns-remediator-task" in AKS Windows nodes which will restart kubeproxy automatically when HNS service crashes.
+* Component Updates
+  * Upgrade Azure Disk V2 CSI Driver to [v2.0.0-beta.6](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v2.0.0-beta.6)
+  * Upgrade Azure Disk CSI driver to [v1.22.0](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v1.22.0)
+  * Azure Monitor for container insights addon updated for Windows to [win-ciprod08102022](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md#08102022--)
```
Comment (shashankbarsin) [inline CHANGELOG.md:26]: this is repeated from previous week's notes?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+  * AKS Ubuntu 18.04 image updated to [AKSUbuntu-1804-2022.08.15](../AKS/vhd-notes/aks-ubuntu/AKSUbuntu-1804/2022.08.15.txt)
+  * AKS Windows 2019 image has been updated to [17763.3287.220810](../AKS/vhd-notes/AKSWindows/2019/17763.3287.220810.txt)
+  * AKS Windows 2022 image has been updated to [20348.887.220810](../AKS/vhd-notes/AKSWindows/2022/20348.887.220810.txt)
```
Comment (shashankbarsin) [inline CHANGELOG.md:29]: these are repeated from previous week

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+* The Open Service Mesh addon has been updated from version 1.1.1 to version [1.2.0](https://github.com/openservicemesh/osm/releases/tag/v1.2.0) for AKS clusters running 1.24.0+. Please note the breaking changes mentioned in the [version 1.2.0 release notes](https://github.com/openservicemesh/osm/releases/tag/v1.2.0)
+
+### Release notes
+
+* Bug fixes
+  * v20220724 contains a bug that will cause clusters running the Open Service Mesh addon to fail when upgrading to v1.24.0, and therefore the upgrade will fail as well. This bug was hotfixed in the v20220731 release. It was also fixed in the v20220807 release but the hotfix needs to rollout to customers already seeing the bug in the mean time. 
+  * Missing CWD(Current Working Directory) field in process creation events fixed. Update low level collector image version from 1.3.42 to 1.3.49
```
Comment (shashankbarsin) [inline CHANGELOG.md:21]: is low level collector image a user facing component? if not, we should remove it from release notes

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,33 @@
...
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+* The Open Service Mesh addon has been updated from version 1.1.1 to version [1.2.0](https://github.com/openservicemesh/osm/releases/tag/v1.2.0) for AKS clusters running 1.24.0+. Please note the breaking changes mentioned in the [version 1.2.0 release notes](https://github.com/openservicemesh/osm/releases/tag/v1.2.0)
+
+### Release notes
+
+* Bug fixes
+  * v20220724 contains a bug that will cause clusters running the Open Service Mesh addon to fail when upgrading to v1.24.0, and therefore the upgrade will fail as well. This bug was hotfixed in the v20220731 release. It was also fixed in the v20220807 release but the hotfix needs to rollout to customers already seeing the bug in the mean time.
```
Comment (shashankbarsin) [inline CHANGELOG.md:20]: so which release was the fix actually in? 2022-08-07? if yes, we should move this point there and update the release description to capture the same. on rollout of this specifically, may be we call it out under announcement section? (as the fix itself was from a previous release and not in this one)

Comment (kaarthis) [reply CHANGELOG.md:20]: @allyford what is the date for SOv cloud and the Airgap as we mention usually at the very top ?

Comment (kaarthis) [reply CHANGELOG.md:20]: On the deprecation of K8s 1.21 version should we add any action for customers as well ?

File: CHANGELOG.md (hunk 3 above)
Comment (allyford) [reply CHANGELOG.md:21]: Yes - confirmed by Robbie Zhang that it's part of the defender component

File: CHANGELOG.md (hunk 1 above)
Comment (allyford) [reply CHANGELOG.md:26]: Removed

File: CHANGELOG.md (hunk 2 above)
Comment (allyford) [reply CHANGELOG.md:29]: Removed

File: CHANGELOG.md (hunk 4 above)
Comment (allyford) [reply CHANGELOG.md:20]: > so which release was the fix actually in? 2022-08-07? if yes, we should move this point there and update the release description to capture the same. on rollout of this specifically, may be we call it out under announcement section? (as the fix itself was from a previous release and not in this one)

Update from engineer - the bug was fixed and hotfixed so there are no versions currently with the bug. Removing as per recommendation

Comment (allyford) [reply CHANGELOG.md:20]: > @allyford what is the date for SOv cloud and the Airgap as we mention usually at the very top ?

As of the previous release notes, we are referencing the AKS Release tracker instead

Comment (allyford) [reply CHANGELOG.md:20]: > On the deprecation of K8s 1.21 version should we add any action for customers as well ?

Adding documentation link for upgrade

**Review by kaarthis — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,30 @@
...
+## Release 2022-08-21
+
+Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks/release-tracker).
+
+### Announcements
+
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
+  * Windows Server 2022 will be the default Windows host. Important, old windows 2019 containers will not work on windows server 2022 hosts.
+* Kubernetes 1.21 version has been deprecated as of July 31st, 2022. See [documentation](https://docs.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli) on how to upgrade your cluster.
```
Comment (shashankbarsin) [inline CHANGELOG.md:12]:
```suggestion
* Kubernetes 1.21 version has been deprecated as of July 31st, 2022. See [documentation](https://docs.microsoft.com/azure/aks/upgrade-cluster?tabs=azure-cli) on how to upgrade your cluster.
```

**Review by shashankbarsin — APPROVED** [review]

**Pull request merged**